projects/commonwealth/index.js

```diff
@@ -0,0 +1,26 @@
+const { staking } = require('../helper/staking');
+
+const TREASURY = [
+  '0xdE70B8BC5215BdF03f839BB8cD0F639D4E3E2881',
```

Comment:
hi @samswealth thanks for the PR; we dont track vesting atm. This treasury is project treasury or user funds that they can withdraw at any point?

Reply:
hi @g1nt0ki, project treasury.